CRM/Event/Form/SelfSvcTransfer.php

```diff
@@ -475,7 +476,7 @@ public function transferParticipantRegistration($toContactID, $fromParticipantID
     ])['values'];
     unset($toParticipantValues['id']);
     $toParticipantValues['contact_id'] = $toContactID;
-    $toParticipantValues['status_id'] = CRM_Core_PseudoConstant::getKey('CRM_Event_BAO_Participant', 'status_id', 'Registered');
+    $toParticipantValues['status_id'] = $statusID;
```

Comment:
@larssandergreen we could also just remove this line couldn't we? - I suspect it could have been required by create in the past but hopefully not now

Reply:
You're right, much better without.

Reply:
Should resolve the test failure too.